### DISCUSSION TITLE
convert a string to phone format?

### DISCUSSION THREAD

**Discussion by Musteri1** [typesnippet/antd-phone-input#103] [Q&A] [answered]
Is it possible to convert a string to phone format?
like : +9053200000 to +90 532 000 0000
or without input text filed like this <PhoneInput value={phone}  disableDropdown={true} just_text_without_country_flug={true}  />
best regards

**Comment by ArtyomVancyan** [comment]
Yes it's possible. Use `getFormattedNumber` function of [typesnippet/react-phone-hooks](https://github.com/typesnippet/react-phone-hooks) in the following way.

```javascript
import {getFormattedNumber} from "react-phone-hooks";

getFormattedNumber("9053200000"); // +90 (532) 000 0000
getFormattedNumber("9053200000").replace(/[()]/g, ""); // +90 532 000 0000
```

This library is dependant to `react-phone-hooks`, so there is no need to install it if you already have `antd-phone-input` installed.

**Reply by Musteri1** [reply]
thank you very much, 
I have one more question
When the first page opens, the country code appears automatically.
but I want it to be blank
I tried these (defaultValue="" and value="" ) but it didn't work.

```
<Form.Item name="company_mobile" rules={[{ validator }]}>
<PhoneInput  defaultValue=""   value=""    />
</Form.Item>
```

This is my first time writing a project with React, so I apologize for these easy questions.

**Reply by ArtyomVancyan** [reply, marked as answer]
You can provide `country` property, specifying the ISO code of the country, but there is no option to show blank. By default, it sets and shows the country code and flag of a country the user is from.

P.S. If you find the answer useful, please consider marking it as an answer so if others have similar questions, they will find it easily 🙂 For the same reason if you have any other questions not similar to this question, please create a new discussion.
